test/everypolitician/popolo/person_test.rb

```diff
@@ -153,7 +153,9 @@ def test_person_gender
   def test_person_equality_based_on_id
     person1 = Everypolitician::Popolo::Person.new(id: '123', name: 'Bob')
     person2 = Everypolitician::Popolo::Person.new(id: '123', name: 'Bob', gender: 'male')
+    organization = Everypolitician::Popolo::Organization.new(id: '123')
     assert_equal person1, person2
+    refute_equal person1, organization
```

Comment:
As a general rule, it's best to only have one expectation per test, unless the later ones are entirely dependent on the prior ones. This is testing a conceptually new thing, so best to isolate it to a new test method.